apps/admin/backend/src/app.test.ts

```diff
@@ -260,7 +252,7 @@ test('getSystemSettings happy path', async () => {
 
   const systemSettingsResult = await apiClient.getSystemSettings();
   assert(systemSettingsResult);
-  expect(systemSettingsResult.arePollWorkerCardPinsEnabled).toEqual(true);
+  expect(systemSettingsResult).toEqual(DEFAULT_SYSTEM_SETTINGS);
```

Comment:
I wonder if it makes sense to parse the system settings loaded on L241 and compare to that, instead of `DEFAULT_SYSTEM_SETTINGS`? If `electionMinimalExhaustiveSampleFixtures` uses a non-default system settings file, then the test will break even though the intent of this test is to make sure what's being read in L241 is correctly parsed, stored in the db, and retrieved in L253

Reply:
Really good call! https://github.com/votingworks/vxsuite/pull/3385/commits/9eadaa6f156a0b73c580e85b02e61699c6c7ea84